<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QProgressBar" name="progressBar" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" >
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>40</y>
      <width>103</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Ping</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="readOnly" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>178</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
